demos/classification_benchmark_demo/cpp_gapi/classification_benchmark_demo_gapi.hpp

```diff
@@ -0,0 +1,110 @@
+static const char reverse_input_channels_message[] = "Optional. Switch the input channels order from BGR to RGB.";
+static const char mean_values_message[] =
+    "Optional. Normalize input by subtracting the mean values per channel. Example: \"255.0 255.0 255.0\"";
+static const char scale_values_message[] = "Optional. Divide input by scale values per channel. Division is applied "
+                                           "after mean values subtraction. Example: \"255.0 255.0 255.0\"";
+static const char use_onevpl_message[] = "Optional. Use onevpl video decoding.";
+static const char onevpl_params_message[] = "Optional. Parameters for onevpl video decoding. "
+                                            "OneVPL source can be fine-grained by providing configuration parameters. "
+                                            "Format: <prop name>:<value>,<prop name>:<value> "
+                                            "Several important configuration parameters: "
+                                            "-mfxImplDescription.mfxDecoderDescription.decoder.CodecID "
+                                            "values: https://spec.oneapi.io/onevpl/2.7.0/API_ref/VPL_enums.html?highlight=mfx_codec_hevc#codecformatfourcc"
+                                            "-mfxImplDescription.AccelerationMode "
+                                            "values: https://spec.oneapi.io/onevpl/2.7.0/API_ref/VPL_disp_api_enum.html?highlight=d3d11#mfxaccelerationmode"
+                                            "(see `MFXSetConfigFilterProperty` by https://spec.oneapi.io/versions/latest/elements/oneVPL/source/index.html)";
+static const char onevpl_pool_size_message[] = "OneVPL source applies this parameter as preallocated frames pool size. 0 leaves frames pool size default for your system. "
+                                               "This parameter doesn't have a god default value. It must be adjusted for specific execution (video, model, system, ...).";
+
+
+DEFINE_bool(h, false, help_message);
+DEFINE_string(m, "", model_message);
+DEFINE_string(labels, "", labels_message);
+DEFINE_string(layout, "", layout_message);
+DEFINE_string(gt, "", gt_message);
+DEFINE_string(d, "CPU", target_device_message);
+DEFINE_uint32(nthreads, 0, num_threads_message);
+DEFINE_string(nstreams, "", num_streams_message);
+DEFINE_uint32(nireq, 1, num_inf_req_message);
+DEFINE_uint32(nt, 5, ntop_message);
+DEFINE_string(res, "1280x720", image_grid_resolution_message);
+DEFINE_bool(auto_resize, false, input_resizable_message);
+DEFINE_bool(no_show, false, no_show_message);
+DEFINE_uint32(time, std::numeric_limits<gflags::uint32>::max(), execution_time_message);
+DEFINE_string(u, "", utilization_monitors_message);
+DEFINE_bool(reverse_input_channels, false, reverse_input_channels_message);
+DEFINE_string(mean_values, "", mean_values_message);
+DEFINE_string(scale_values, "", scale_values_message);
```

Comment:
These, `layout`, `auto_resize` are ignored in your implementation

Reply:
These three still exist: `reverse_input_channels`, `mean_values`, `scale_values`